src/Setup/InstallSchema.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EasyTranslate\CompatMageplazaBlog\Setup;
+
+use Magento\Framework\DB\Ddl\Table;
+use Magento\Framework\Setup\InstallSchemaInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+use Zend_Db_Exception;
+
+class InstallSchema implements InstallSchemaInterface
+{
+    /**
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     * @throws Zend_Db_Exception
+     */
+    public function install(SchemaSetupInterface $setup, ModuleContextInterface $context): void
+    {
+        $setup->startSetup();
```

Comment:
Is there a specific reason why you call `startSetup` and `endSetup`? If there is none, please remove it. It disables foreign keys and that is usually not what we want.